tests/form_elements.py

```diff
@@ -89,6 +89,13 @@ def test_clicking_submit_button_posts_button_value_if_value_present(self):
             "submit-button: submit-button-value",
         )
 
+    def test_submitting_a_form_when_the_button_is_outside_the_form(self):
+        self.browser.find_by_css('input[form="form-with-submit-outside"]').click()
+        self.assertEqual(
```

Comment:
For new tests, it's better to use the pytest syntax:
```
assert "submit-input: submit-input-value" == self.browser.find_by_xpath("/descendant-or-self::*").text
```

Reply:
Thanks, updated.